controller/src/main/java/com/softserve/borda/controllers/UserController.java

```diff
@@ -108,19 +95,21 @@
         }
     }
 
-    @PostMapping("update/{id}")
-    public UserSimpleDTO updateUser(@RequestBody final UserUpdateDTO userDTO, @PathVariable String id) {
-        var user = jwtConvertor.getUserByJWT(id);
+    @PostMapping("update")
```

Comment:
Bad resource naming.
Please read https://restfulapi.net/resource-naming/ doc and change naming.

Reply:
Fixed in FixResourcesPaths branch/ticket, already merged in main